lib/Runtime/SerializableFunctionFields.h

```diff
@@ -71,6 +71,7 @@ PRIVATE_FIELDS
     DECLARE_SERIALIZABLE_FIELD(bool, m_hasFinally, Bool);
     DECLARE_SERIALIZABLE_FIELD(bool, hasScopeObject, Bool);
     DECLARE_SERIALIZABLE_FIELD(bool, hasCachedScopePropIds, Bool);
+    DECLARE_SERIALIZABLE_ACCESSOR_FIELD(uint, ForInLoopDepth, UInt32);
```

Comment:
Oh ok. Thanks

---
In reply to: [81579788](https://github.com/Microsoft/ChakraCore/pull/1641#discussion_r81579788) [](ancestors = 81579788)